api/vertex_test.go

```diff
@@ -109,6 +109,24 @@ func TestHasLabel(t *testing.T) {
 	assert.Equal(t, fmt.Sprintf("%s.V().hasLabel(\"%s\")", graphName, label), v.String())
 }
 
+func TestHasLabelMulti(t *testing.T) {
+	// GIVEN
+	graphName := "mygraph"
+	g := NewGraph(graphName)
+	require.NotNil(t, g)
+	v := g.V()
+	require.NotNil(t, v)
+	l1 := "label1"
+	l2 := "label2"
```

Comment:
What if we also had a test coverage for a given array. for instance 
```
labels := []string{"label1","label2"}
v = v.HasLabel(labels...)
```
if the `v = v.HasLabel(l1,l2)` works, the above example should also pass, but I just thought it would be worth testing both cases

Reply:
We'd just test basic golang features with using an array that gets unpacked, so I'd leave it as it is. It makes no difference.
If we'd wanna test more than 2 labels, then of course the array would make more sense.

Reply:
I agree to @dArignac as you can see on the [golang spec](https://golang.org/ref/spec#Passing_arguments_to_..._parameters) for variadic functions parameters like `t ...T` are represented as an array inside the function (`t []T`).

So calling the function with `f("a","b","c")` or `f(myStrArray...)` makes no difference inside the function we want to test. Both variables "arrive" inside the function as an array anyway.